tests/tools/test_dist_part.py

```diff
@@ -195,20 +55,32 @@ def rand_edges(num_src, num_dst, num_edges):
         with open(meta_fname, 'rb') as f:
             meta_data = json.load(f)
 
-        all_etypes = ['affiliated_with', 'writes', 'cites', 'rev_writes']
+        #all_etypes = ['affiliated_with', 'writes', 'cites', 'rev_writes']
         for etype in all_etypes:
             assert len(meta_data['edge_map'][etype]) == num_parts
         assert meta_data['etypes'].keys() == set(all_etypes)
         assert meta_data['graph_name'] == 'mag240m'
 
-        all_ntypes = ['author', 'institution', 'paper']
+        #all_ntypes = ['author', 'institution', 'paper']
         for ntype in all_ntypes:
             assert len(meta_data['node_map'][ntype]) == num_parts
         assert meta_data['ntypes'].keys() == set(all_ntypes)
         assert meta_data['num_edges'] == g.num_edges()
         assert meta_data['num_nodes'] == g.num_nodes()
         assert meta_data['num_parts'] == num_parts
 
+        # Create Id Map here.
+        edge_dict = {
```

Comment:
could we avoid define the id ranges manually? could we utilize dgl methods to obtain this?

Reply:
Done. Removed hardcoded values. Now we use the underlying graph to build this dictionary.